src/screens/Onboarding/Onboarding.stories.tsx

```diff
@@ -1,39 +1,28 @@
-import { action } from "@storybook/addon-actions";
 import type { Meta, StoryObj } from "@storybook/react";
-import { findByRole, userEvent } from "@storybook/testing-library";
-import { rest } from "msw";
+import { graphql } from "msw";
 
 import { storyWrapper } from "../../utils/graphQLClient";
-import { playAll } from "../../utils/playAll";
+import { withFigmaDesign } from "../../utils/withFigmaDesign";
 import { Onboarding } from "./Onboarding";
 
 const meta = {
   component: Onboarding,
   decorators: [storyWrapper],
-  args: {
-    setAccessToken: action("setAccessToken"),
-  },
   parameters: {
     msw: {
       handlers: [
-        rest.post("*/authorize", (req, res, ctx) =>
+        graphql.query("ProjectQuery", (req, res, ctx) =>
           res(
-            ctx.json({
-              device_code: "chdc_95a7123d17a84851abcdefc869ec0741",
-              user_code: "123 123",
-              verification_uri: "https://www.chromatic.com/connect/storybook-visual-tests",
-              verification_uri_complete:
-                "https://www.chromatic.com/connect/storybook-visual-tests?code=123123",
-              expires_in: 300,
-              interval: 5,
-            })
-          )
-        ),
-        rest.post("*/token", (req, res, ctx) =>
-          res(
-            ctx.json({
-              error: "authorization_pending",
-              error_description: "Authorization is pending approval",
+            ctx.data({
+              project: {
```

Comment:
Similarly, is there a way to type `ctx.data()` here?

Reply:
Looks like this works: 
<img width="584" alt="image" src="https://github.com/chromaui/addon-visual-tests/assets/132554/01c14e16-2bd1-4cdc-85f0-86c6c9301863">

Or we can do it semi automatically with https://the-guild.dev/graphql/codegen/plugins/typescript/typescript-msw (although I'm not sure there's a huge benefit)